src/project-3.js

```diff
@@ -1,75 +1,134 @@
+  //  obj.age = age;
+  //  obj.meow = function(){...};
+  //
+  const newObject = {
+    name,
+    age,
+    meow: () => {
+      return 'Meow!';
+    }
+  };
+
+  return newObject;
 };
 
 const addProperty = (object, property) => {
   // add the property to the object with a value of null
   // return the object
   // note: the property name is NOT 'property'.  The name is the value of the argument called property (a string)
+  /*
+   * in: object, property
+   * +in: {nationality: 'Canadian'}, "name"
+   * out: {name: null, nationality: 'Canadian'}
+   */
+  object[property] = null;
+  return object;
 };
 
 const invokeMethod = (object, method) => {
   // method is a string that contains the name of a method on the object
   // invoke this method
   // nothing needs to be returned
+  object[method]();
 };
 
 const multiplyMysteryNumberByFive = (mysteryNumberObject) => {
   // mysteryNumberObject has a property called mysteryNumber
   // multiply the mysteryNumber property by 5 and return the product
+  const product = mysteryNumberObject.mysteryNumber * 5;
+  return product;
 };
 
 const deleteProperty = (object, property) => {
   // remove the property from the object
   // return the object
+  delete object[property];
+  return object;
 };
 
 const newUser = (name, email, password) => {
+  return { name, email, password };
+};
   // create a new object with properties matching the arguments passed in.
   // return the new object
-};
 
-const hasEmail = (user) => {
+const hasEmail = user => !!user.email;
   // return true if the user has a value for the property 'email'
   // otherwise return false
-};
 
-const hasProperty = (object, property) => {
+const hasProperty = (object, property) => !!object[property];
   // return true if the object has the value of the property argument
   // property is a string
   // otherwise return false
-};
 
-const verifyPassword = (user, password) => {
+const verifyPassword = (user, password) => user.password === password;
   // check to see if the provided password matches the password property on the user object
   // return true if they match
   // otherwise return false
-};
 
 const updatePassword = (user, newPassword) => {
+  user.password = newPassword;
+  return user;
+};
   // replace the existing password on the user object with the value of newPassword
   // return the object
-};
+
 
 const addFriend = (user, newFriend) => {
+  user.friends.push(newFriend);
+  return user;
   // user has a property called friends that is an array
   // add newFriend to the end of the friends array
   // return the user object
 };
 
 const setUsersToPremium = (users) => {
+  for (let i = 0; i < users.length; i++) {
+    users[i].isPremium = true;
+  }
+  return users;
   // users is an array of user objects.
   // each user object has the property 'isPremium'
   // set each user's isPremium property to true
   // return the users array
 };
 
 const sumUserPostLikes = (user) => {
+  let sum = 0;
```

Comment:
`reduce` would work here as well.  In the future you'll want to use `map`, `forEach`, and `reduce` whenever possible.  This current solution is fine but in the future you'll want to solve these differently :)